### ISSUE TITLE
Better way to prohibit changes to multiple targets

### ISSUE THREAD

**Issue by pbloigu** [Azure/application-gateway-kubernetes-ingress#928]
**Is your feature request related to a problem? Please describe.**
We have 3 clusters behind an app gw. Each cluster has multiple hosts exposed. For each cluster I now have to create ProhobitedTargets for each of the hostnames in the other two clusters. This is a bit cumbersome.

**Describe the solution you'd like**
I would be fine with either of the two solutions:
1. Allow for automatically prohibiting changes to hostnames not present in the current cluster
2. Allow specifying multiple hostnames for a ProhibitedTarget, preferably with a regular expression

**Comment by 3quanfeng** [comment]
Hi @pbloigu , are all of your 3 clusters are AKS clusters?

**Comment by pbloigu** [comment]
> Hi @pbloigu , are all of your 3 clusters are AKS clusters?

Yes they are. Do you have a better approach in mind to achieve what I described that already works with AGIC as it is? I would love to hear about it as I'm in the midst setting up the environments right now. Changing the environment later on will be harder as they're in active use.

**Comment by 3quanfeng** [comment]
No, you will have to configure the prohibited policy on each of the cluster, or you could use one appgw per cluster without any prohibited targets?

**Comment by derkoe** [comment]
We want to use one application gateway for multiple teams/AKS clusters. One team does not know which URLs the other teams use. For this use case and generally for an easier configuration it would be great to have `AzureIngressAllowedTarget`s with wildcards that can be defined per cluster.

**Comment by aelmanaa** [comment]
Hello,

any milestone to deliver this feature? we have exactly the same need as @pbloigu

**Comment by swketechie-mtn-gh** [comment]
hello @Azure 
Is there any plan on delivering this functionality? 
or
has it been delivered already? if yes, please direct to link for ref

Gracias

**Comment by umeshjadhav-tss** [comment]
We're still suffering with the same , I think azure should take responsible action on this .
At least a **wildcard** entry support should be there to save efforts and risk.  
Sometimes cloud team forgets to add the entries and it gets deleted by AGIC and it affects the production environment.
@Azure please provide ref if any update available or developed in the market 
**Thanks,
 Umesh**
